WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -1862,7 +1862,7 @@ public void testAnimalHistory()
                 .clickCategoryTab("Clinical")
                 .clickReportTab("Weights");
         waitForElement(Locator.xpath("//span[contains(text(), 'Weight Information -')]"));
-        waitForElement(Locator.tagContainingText("th", "Most Recent Weight").notHidden());
+        waitForElement(Locator.tag("th").containing("Most Recent Weight").notHidden());
```

Comment:
Would there be any value in changing tagContainingText to be more like containing (in it's comparison check)?

Reply:
The comment on `tagContainingText` implies I tried that at some point and it didn't go well.